src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -11,8 +11,8 @@ import cromwell.engine.db.DataAccess._
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.WorkflowActor.{Restart, Start}
+import cromwell.engine.workflow.WorkflowManagerActor.{WorkflowManagerData, WorkflowManagerState}
```

Comment:
I personally strongly prefer having just about everything up at the top unless there's a good reason not to (e.g. pulling a specific implicit into scope)